.github/workflows/reusable-build.yml

```diff
@@ -167,6 +167,39 @@ jobs:
             # we can retry on that unfortunately common failure case
             podman pull quay.io/${{ env.SOURCE_ORG }}/${{ env.SOURCE_IMAGE }}:${{ matrix.fedora_version }}
 
+      # buildah on ubuntu-22.04 builder is pretty dated and the version on
+      # ubuntu-24.04 misbehaves when using heredocs in Containerfiles
+      # (as of 2024-07-12)
+      # this wrapper script mimics toolbx/distrobox behaviour for buildah which
+      # is invoked by redhat-actions/buildah-build@v2
+      - name: Install wrapper script for containerized buildah
+        id: fix-gh-buildah
+        shell: bash
+        run: |
+          cat > /usr/local/bin/buildah <<'EOF'
+            #!/bin/sh
+            set -eux
+
+            # get local graphroot at runtime
+            GRAPH_ROOT="$(/usr/bin/buildah info | jq -r '.store.GraphRoot')"
+
+            exec podman run --rm \
+              --privileged \
+              --net=host \
+              --cgroups=disabled \
+              --runtime=crun \
+              --runtime-flag=cgroup-manager=disabled \
+              --security-opt=label=disable \
+              --security-opt=seccomp=unconfined \
+              --device=/dev/fuse:rw \
+              -v /workspace:/workspace \
```

Comment:
I'm guessing this is a difference between the gitea local runner and githubs runner. I'm assuming this is supposed to be the current directory.

Reply:
I guess you're right. It looks like the runners home directory is the github-equivalent.